### ISSUE TITLE
context must be a dict rather than RequestContext

### ISSUE THREAD

**Issue by henryjameslau** [overcastsoftware/wagtailcharts#19]
I'm trying to get a blank version of wagtail working with wagtailcharts but I keep getting an error

<h1>TypeError
       at /</h1>
  <pre class="exception_value">context must be a dict rather than RequestContext.</pre>

<h2>Error during template rendering</h2>
   <p>In template <code>/code/wagtailcharts_demo/home/templates/home/home_page.html</code>, error at line <strong>23</strong></p>
   <h3>context must be a dict rather than RequestContext.</h3>
   
```
13 | </div>
-- | --
{% for block in self.specific.body %}
{% include_block block %}
{% endfor %}
</div>
{% endblock content %}
 
 
{% block extra_js %}
<script src="{% static 'js/main.js' %}"></script>
{% render_charts %}
{% endblock %}
 
```
 I've tried setting it up through the command line using your repo and with docker but no success. Any pointers please?

**Comment by saevarom** [comment]
Hi @henryjameslau 

To clarify, which version of Wagtail?

**Comment by henryjameslau** [comment]
I'm using what's set out in your [requirements](https://github.com/overcastsoftware/wagtailcharts/blob/main/wagtailcharts_demo/requirements.txt) so wagtail 6.2.1

**Comment by saevarom** [comment]
Right, we might need to do more pinning between minor versions of wagtail. 
Does the same happen if you do `pip install wagtail==6.1.3` and try again?

**Comment by saevarom** [comment]
Nevermind, confirmed it myself

**Comment by saevarom** [comment]
Fixed in release [0.6.1](https://pypi.org/project/wagtailcharts/0.6.1/)
